docs/source/api/timer.rst

```diff
@@ -0,0 +1,377 @@
+##########
+Timer
+##########
+
+About
+-----
+
+The ESP32 SoCs contains from 2 to 4 hardware timers. 
+They are all 64-bit (54-bit for ESP32-C3) generic timers based on 16-bit pre-scalers and 64-bit (54-bit for ESP32-C3) 
+up / down counters which are capable of being auto-reloaded.
+
+========= ================
+ESP32 SoC Number of timers
+========= ================
+ESP32     4
+ESP32-S2  4
+ESP32-C3  2
+ESP32-S3  4
+========= ================
+
+Arduino-ESP32 Timer API
+----------------------------
+
+timerBegin
+**********
+
+This function is used to configure the timer. After successful setup the timer will automatically start.
+
+.. code-block:: arduino
+
+    hw_timer_t * timerBegin(uint8_t num, uint16_t divider, bool countUp);
+
+* ``num`` select timer number.
+* ``divider`` select timer divider.
+* ``resolution`` select timer resolution.
+
+  * range is 1-14 bits (1-20 bits for ESP32).
```

Comment:
Uhhm, no...the 3rd parameter is a `bool` called `countUp`.

Reply:
Thanks @pedrominatel for taking care of it. Sorry for the mistake.